post-process.py

```diff
@@ -0,0 +1,123 @@
+import os
+import pandas as pd
+from io import StringIO as sio
+import re
+import sys
+import argparse
+from pathlib import Path
+import glob
+
+""" Global Variables """
+
+file_name = "vernier-output-"
```

Comment:
This should be a command line argument as well, probably something like input file base name or similar. We might have to make the script a bit cleverer in future to deal with various options of the file name.